capstone/src/main/webapp/scripts/poll.js

```diff
@@ -10,12 +11,39 @@ function getPollOptions() {
     .then(options => {
       const optionsContainer = document.getElementById("options-container");
       optionsContainer.innerHTML = "";
-      options.forEach(option => {
+      options["options"].forEach(option => {
```

Comment:
If you are getting the "options" from options, does this variable perhaps represent something else other than an options?

Reply:
In the JSON string I have a field that's called options, which contains options data, so I have to index into that

Reply:
Thank you, pollData makes more sense and makes it more clear 👍